docker/compose.sls

```diff
@@ -1,10 +1,6 @@
 {% from "docker/map.jinja" import compose with context %}
 
 compose-pip-dependencies:
-  pkg.installed:
-    - name: python-pip
   pip.installed:
     - name: docker-compose == {{ compose.version }}
-    - require:
-      - pkg: python-pip
     - reload_modules: true
```

Comment:
Why reloading modules ? Is there a docker salt execution modules available only when docker-compose is installed ?